Source/HelixToolkit.Tests/HelixToolkit.Tests.csproj

```diff
@@ -57,7 +57,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="NUnit">
-      <Version>3.10.1</Version>
+      <Version>3.12.0</Version>

```

Comment:
The   `[Apartment(ApartmentState.STA)]` did not exist in the .NET Core version 3.10.1 so I upgraded all to 3.12.0 for consistency.